pkg/cmd/factory/default_test.go

```diff
@@ -362,9 +363,25 @@ func Test_ioStreams_prompt(t *testing.T) {
 			config:         disablePromptConfig(),
 			promptDisabled: true,
 		},
+		{
```

Comment:
looking through the code this seemed like the only place to add to the tests, please let me know if I missed others!